libraries/AC_AttitudeControl/AC_AttitudeControl_Multi.cpp

```diff
@@ -439,35 +439,49 @@ void AC_AttitudeControl_Multi::update_throttle_rpy_mix()
     _throttle_rpy_mix = constrain_float(_throttle_rpy_mix, 0.1f, AC_ATTITUDE_CONTROL_MAX);
 }
 
-void AC_AttitudeControl_Multi::rate_controller_run()
+void AC_AttitudeControl_Multi::rate_controller_run_dt(float dt, const Vector3f& gyro)
```

Comment:
it doesn't really matter but normally we put "dt" as the last argument

Reply:
Thanks, fixed